app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,9 @@
+class DashboardController < ApplicationController
+
+  def feedback
+    # wishlists ---> wishes ---> comments
+    organization = current_user.organization
```

Comment:
What if we are admins visiting another page organization? Maybe we should nest the feedback dashboard under organization and get the organization ID through the routes?

Reply:
suree makes sense

Reply:
If i do what suggested we will loose this:
```
authenticated(:user) do
  root to: "dashboard#feedback", as: :authenticated_root
end
```
The link will have to include `organizations/organizations_id/dashboard...` to retrieve the `organization_id`

What do you think? Any solution? I guess being able to get the `organization_id` more important but...

Also....

The policy is hard to build because dashboard is not an ActiveModel so it does not have an ActiveRelation with organizations and I can not to something like `user.organization == record.organization` or similar.

Alternative: Add the `feedback` action to `OrganizationsController` . And later, depending on how much stuff we have in that controller due to the "dashboard" actions we think of refactoring it.

LMK what you think? 🙃

Reply:
Yes I think we have to move each page pf the dashboard as a "member" or "collection" route of each controller, in this case, feedback will be a collection route of the organization's controller?

Reply:
yep! let's add this actions into collections of each controller and once every action is settled we can decide on a refactor. It solves all the issues.